### ISSUE TITLE
Encoder not transferring  file after encoding 

### ISSUE THREAD

**Issue by barrunning** [WWBN/AVideo#2658]
Encoder uploads and encodes video but doesn’t transfer it to website. 
No error message are shown. 
Plugins are up to date and encoder is up to date.

**Comment by DanielnetoDotCom** [comment]
please check your log files, for encoder and streamer

**Comment by wahabzafar** [comment]
issue is there in public encoder anyone's file is stuck on it so now whoever upload through encoder files goes pending unable to find any solution.

**Comment by maxlinux2000** [comment]
have a look to the domain name you are login in the encoder.
I had a similar fault, and in the end it was simply that in the sendrecordtoencoder plugin I had a url with a different url from where the streamer was.

The streamer was at https://s1.mysite.com, but in the parameters of the plugin it had put https://mysite.com

**Comment by barrunning** [comment]
So how do I check that and how do I change it? I know in configuration it was set to public encoder and not private. I switch that to private url

**Comment by barrunning** [comment]
I am using a private encoder not the public one. It work fine for one video and then it stopped working. I am trying to figure out why it worked fine for one. How do I check logs to see what happened? I did not install this danieldotcom set it up for me so I am in the dark about where to start looking. It is installed on aws s3.  I had been using the public encoder when the public encoder stopped working. danieldotcom then said I should not be using the public encoder but my private one. I tried my private encoder and it worked. The next video I try to upload it encoded but didn't transfer. This was like 2 minutes later.
